pandas/tseries/resample.py

```diff
@@ -673,18 +670,15 @@ def aggregate(self, arg, *args, **kwargs):
     def _get_new_index(self):
         """ return our new index """
         ax = self.ax
-        ax_attrs = ax._get_attributes_dict()
-        ax_attrs['freq'] = self.freq
-        obj = self._selected_obj
 
         if len(ax) == 0:
-            new_index = PeriodIndex(data=[], **ax_attrs)
-            return obj.reindex(new_index)
-
-        start = ax[0].asfreq(self.freq, how=self.convention)
-        end = ax[-1].asfreq(self.freq, how='end')
+            values = []
+        else:
+            start = ax[0].asfreq(self.freq, how=self.convention)
+            end = ax[-1].asfreq(self.freq, how='end')
```

Comment:
ahh, so this should actually be returning a ``PeriodIndex`` where freq is 5D, then the ``convention`` wouldn't matter as we are not going to a DTI. hmm. I think we need to be more explicit about returning ``PeriodIndex`` then, right?